src/models/github.js

```diff
@@ -11,6 +11,22 @@ class GitHub extends Fetcher {
     return this.get('notifications')
   }
 
+  getPandaRocketTasks() {
+    const urlPath = 'search/issues?q=team=identity-team-panda-rocket'
+    return this.get(urlPath).then(({ items }) => {
+      return items.map(item => {
+        return {
+          title: item.title,
+          body: item.body,
+          state: item.state,
+          createdAt: new Date(item.created_at),
```

Comment:
Yeah, I was going to make it automatic, but we only need a few fields so I did it manually.